src/fastfield/alive_bitset.rs

```diff
@@ -7,6 +6,22 @@ use common::ReadSerializedBitSet;
 use std::io;
 use std::io::Write;
 
+/// Merges (intersects) two AliveBitSet in a new one.
+pub fn merge_alive_bitset(left: &AliveBitSet, right: &AliveBitSet) -> crate::Result<AliveBitSet> {
```

Comment:
I'd rather have an assert on the max_doc() equality. (and add a max_doc on the AliveBitset)

Reply:
Rename intersects.

Merge is used for union and intersection, which makes it very ambiguous.

Reply:
yep that's better